CHANGELOG.md

```diff
@@ -1,5 +1,80 @@
 # Changelog
 
+## v6.0.0-beta.7 - 2017-09-13
+
+### Breaking change
+
+* Fix `Get-Content -Delimiter` to not include the delimiter in the array elements returned (#3706) (Thanks @mklement0)
+
+### Engine updates and fixes
+
+* Use stricter rules when unwrapping a PSObject that wraps a COM object (#4614)
+* Remove appended Windows PowerShell `PSModulePath` on Windows. (#4656)
+* Ensure `GetNetworkCredential()` returns null if PSCredential has null or empty user name (#4697)
+* Push locals of automatic variables to 'DottedScopes' when dotting script cmdlets (#4709)
+* Fix `using module` when module has non-terminating errors handled with `SilentlyContinue` (#4711) (Thanks @iSazonov)
+* Enable use of 'Singleline,Multiline' option in split operator (#4721) (Thanks @iSazonov)
+* Fix error message in `ValidateSetAttribute.ValidateElement()` (#4722) (Thanks @iSazonov)
+* Rename $IsOSX to $IsMacOS (#4757)
```

Comment:
We should call this out as a Breaking Change (from previous PSCore6 release)

Reply:
OK. Should we add the tag to the PR as well?

Reply:
Yes, should have been tagged.  I think the issue was tagged, but not the PR.